performance/src/include/firebase/performance/trace.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_TRACE_H_
 #define FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_TRACE_H_
```

Comment:
⚠️ Lint warning: `Lines should be <= 80 characters long`<hidden value="cpplint-file-comment"></hidden><hidden value="performance/src/include/firebase/performance/trace.h:4:  Lines should be <= 80 characters long  [whitespace/line_length] [2]"></hidden>